octue/cloud/storage/client.py

```diff
@@ -6,21 +6,23 @@
 import os
 import warnings
 
-import google.api_core.exceptions
-import google.auth.exceptions
 from google import auth
+import google.api_core.exceptions
 from google.auth import compute_engine
+import google.auth.exceptions
 from google.auth.transport import requests as google_requests
 from google.cloud.storage import Client
 from google.cloud.storage.constants import _DEFAULT_TIMEOUT
 from google.cloud.storage.retry import DEFAULT_RETRY
-from google_crc32c import Checksum
 
 from octue.cloud import storage
 from octue.exceptions import CloudStorageBucketNotFound
 from octue.utils.decoders import OctueJSONDecoder
 from octue.utils.encoders import OctueJSONEncoder
 
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
+    from google_crc32c import Checksum
```

Comment:
Doesn't this modify the warnings filter to ignore everything that comes after, regardless of the catch? 

Also, I feel like this issue should actually be solved; have you tried updating to later versions of `google_crc32`? Just wondering if the rest of the google client libraries have upgraded their dependencies in the last couplee of years since crc32 began to build their wheels properly.